install_files/ansible-base/roles/build-ossec-deb-pkg/defaults/main.yml

```diff
@@ -1,5 +1,5 @@
 ---
-build_ossec_deb_pkg_dependencies: ['libevent1-dev','libpcre2-dev']
+build_ossec_deb_pkg_dependencies: ['libevent-dev','libpcre2-dev']
```

Comment:
Why is this dependency changed?

Reply:
+1, if we don't have to change this dependency, let's not. @kushaldas can you check whether the `libevent1-dev` is sufficient for both platforms? If not, can you provide documentation that `libevent-dev` is correct for both?

Reply:
This has to updated based on `Focal` or `Xenial`.